src/Catalyst.TestUtils/TransactionHelper.cs

```diff
@@ -67,5 +68,49 @@ public static class TransactionHelper
             };
             return transaction;
         }
+        
+        public static TransactionBroadcast GetContractTransaction(ByteString data,
+            UInt256 amount,
+            uint gasLimit,
+            UInt256 gasPrice,
+            byte[] targetContract = null, // to be reviewed
+            string senderPublicKey = "sender",
+            string receiverPublicKey = "receiver",
+            string signature = "signature",
+            long timestamp = 12345,
+            ulong transactionFees = 2,
+            ulong nonce = 0,
+            NetworkType networkType = NetworkType.Devnet)
+        {
+            var transaction = new TransactionBroadcast
+            {
+                ContractEntries =
+                {
+                    new ContractEntry
+                    {
+                        Amount = ((UInt256) amount).ToUint256ByteString(),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unnecessary cast to 'UInt256'.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4350015)